inc/functions.php

```diff
@@ -15,12 +18,17 @@ function wpcs_add_track_fields() { ?>
 
 // add fields to edit track taxonomy
 function wpcs_edit_track_fields( $term, $taxonomy ) {
-	$value = get_term_meta($term->term_id, 'track_color', true);
+	$backgroundValue = get_term_meta($term->term_id, 'track_background_color', true);
+	$textValue = get_term_meta($term->term_id, 'track_text_color', true);
 	 ?>
 
 	<tr class="form-field">
-		<th scope="row"><label for="track_color">Track Color</label></th>
-			<td><input type="text" name="track_color" id="track_color" class="track-color-picker" size="40" value="<?php echo esc_attr($value); ?>"></td>
+		<th scope="row"><label for="track_background_color">Track Background Color</label></th>
+			<td><input type="text" name="track_background_color" id="track_background_color" class="track-color-picker" size="40" value="<?php echo esc_attr($backgroundValue); ?>"></td>
+	</tr>
+	<tr class="form-field">
+		<th scope="row"><label for="track_text_color">Track Background Color</label></th>
```

Comment:
The label here should be "Track **Text** Color".

Reply:
@kjohnson fixed this in commit [7f0e8a0](https://github.com/stellarwp/wpconferenceschedule/pull/7/commits/7f0e8a0392aba0995a2fde6c985ee6bd40af09ab)